### PR TITLE
Adds support for returning app schema

### PR DESCRIPTION
**Pull request by jsancio**

### CODE REVIEW

**Comment by ConnorDoyle** [comment]
xref #1185

File: src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@Path("v2/schemas")
+@Consumes(Array(MediaType.APPLICATION_JSON))
+@Produces(Array(MediaType.APPLICATION_JSON))
+class SchemaResource @Inject() (
+    val config: MarathonConf) extends RestResource with ModelValidation {
+
+  @GET
+  @Timed
+  def index(): InputStream = {
```
Comment (drexin) [inline src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala]: This should also have `@Path("/app")`

**Comment by drexin** [comment]
LGTM aside from the path. Could you also please squash this into a single commit?

File: src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala (hunk 1 above)
Comment (ssk2) [inline src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala]: +1

Comment (jsancio) [inline src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala]: Good catch! Fill fix.

Comment (jsancio) [inline src/main/scala/mesosphere/marathon/api/v2/SchemaResource.scala]: Done.

**Comment by jsancio** [comment]
Squashed the commits and rebase against the latest master. There is also a formatting change in a unrelated file.

**Comment by drexin** [comment]
:+1:

**Pull request merged**